diff --git a/node_modules/expo-sharing/ios/SharingModule.swift b/node_modules/expo-sharing/ios/SharingModule.swift
index 71c61a1..bcbecda 100644
--- a/node_modules/expo-sharing/ios/SharingModule.swift
+++ b/node_modules/expo-sharing/ios/SharingModule.swift
@@ -14,7 +14,7 @@ public final class SharingModule: Module {
     Name("ExpoSharing")
 
     AsyncFunction("shareAsync") { (url: URL, options: SharingOptions, promise: Promise) in
-      guard FileSystemUtilities.isReadableFile(appContext, url) else {
+      guard url.isFileURL else {
         throw FilePermissionException()
       }
 
